CONTRIBUTING.md

```diff
@@ -1,6 +1,38 @@
 # Contributing
 First of all, thank you for contributing. It’s appreciated. This guide details how to use issues and pull requests to improve this project.
 
+Table of Contents
+=================
+
+  * [Requirements:](#requirements)
+  * [Installation](#installation)
+  * [Running](#running)
+  * [Making changes](#making-changes)
+    * [Branching](#branching)
+    * [Commit message](#commit-message)
+  * [Creating an Issue](#creating-an-issue)
+  * [Tests](#tests)
+
+## Requirements:
+* NodeJS(v4 LTS)
+* git crypt
+
+## Installation
+Clone this repository
+
+```bash
+$ git clone https://github.com/sphereio/sphere-product-type-import.git
+```
+
+We are using git-crypt for encrypting configs.
+Run
+
+```bash
+$ git-crypt unlock
+```
+
+to unlock the config file.
```

Comment:
As soon as we write credentials to a **file** inside the repo, we have the **risk of committing credentials**.
On the other side, if we keep around **environment variables**, we have the risk of accidentally running the tests (locally) with some **production credentials**.

So to avoid both risks we could add a `--projectKey` param to the tests command, that is **always required**. The tests then search for the corresponding credentials in the environment variables. If the `SPHERE_PRODUCT_KEY` env var is different to the provided `--projectKey` the tests throws an error.
That way we avoid using config files, and avoid the risk of accidentally using production credentials 🚀